docs/apache-airflow-providers-amazon/secrets-backends/aws-secrets-manager.rst

```diff
@@ -27,7 +27,7 @@ Here is a sample configuration:
 
     [secrets]
     backend = airflow.providers.amazon.aws.secrets.secrets_manager.SecretsManagerBackend
-    backend_kwargs = {"connections_prefix": "airflow/connections", "variables_prefix": "airflow/variables", "profile_name": "default", "full_url_mode": False}
+    backend_kwargs = {"connections_prefix": "airflow/connections", "variables_prefix": "airflow/variables", "profile_name": "default", "full_url_mode": "False"}
```

Comment:
Isn't backend kwargs parsed as json? So the correct fix for this would be


```suggestion
    backend_kwargs = {"connections_prefix": "airflow/connections", "variables_prefix": "airflow/variables", "profile_name": "default", "full_url_mode": false}
```

Reply:
Yeah, this should work. I don't get what the problem is here,